### PR TITLE
:seedling: TK-2631 : Added missing resources logic on OCP

### PR DESCRIPTION
**Pull request by pallav-trilio**

### CODE REVIEW

File: tools/log-collector/helper.go [hunk 1]
```diff
@@ -71,6 +73,9 @@ func aggregateEvents(eventObjects unstructured.UnstructuredList,
 		}
 
 		namespace, _, nErr := unstructured.NestedString(eve.Object, "involvedObject", "namespace")
+		if namespace == "" {
```
Comment (msurudwad) [inline tools/log-collector/helper.go:76]: This `if` block should be placed after checking `nErr`

File: tools/log-collector/helper.go [hunk 2]
```diff
@@ -208,13 +212,12 @@ func getResourceByName(gVResources []apiv1.APIResource, name string) (matchedRes
 // getContainerStatusValue returns whether current and previous container present to capture logs
 func getContainerStatusValue(containerStatus *corev1.ContainerStatus) (conStatObj containerStat) {
 
-	lastState := containerStatus.LastTerminationState
 	currentState := containerStatus.State
 
-	if lastState.Terminated != nil {
+	if currentState.Running != nil && currentState.Terminated != nil {
```
Comment (msurudwad) [inline tools/log-collector/helper.go:217]: I think, this condition will never be `true` as `currentState` cannot be both `Running` & `Terminated`. Check it once.

File: tools/log-collector/log_collector.go [hunk 3]
```diff
@@ -340,6 +332,9 @@ func (l *LogCollector) writeLogs(resourceDir string, obj unstructured.Unstructur
 	var podObj corev1.Pod
 	err := l.k8sClient.Get(context.Background(), types.NamespacedName{Name: objName, Namespace: objNs}, &podObj)
 	if err != nil {
+		if errors.IsNotFound(err) {
```
Comment (msurudwad) [inline tools/log-collector/log_collector.go:335]: Log this err... before returing `nil`

**Review by msurudwad — CHANGES_REQUESTED** [review]
- Set LogLevel for `cmd` package as well and it should set before actual execution starts. Make changes to `logcollector` for this.

- Change `doc`as per changes made in this PR. Currently, all resources which are collected by log-collector are not mentioned in doc. Review it.

File: tools/log-collector/helper.go (hunk 1 above)
Comment (pallav-trilio) [reply tools/log-collector/helper.go:76]: done

File: tools/log-collector/log_collector.go (hunk 3 above)
Comment (pallav-trilio) [reply tools/log-collector/log_collector.go:335]: done

File: tools/log-collector/helper.go (hunk 2 above)
Comment (pallav-trilio) [reply tools/log-collector/helper.go]: done

**Review by msurudwad — COMMENTED** [review]
LGTM.. resolve other review comments as part of another PR

**Review by madhurtrilio — APPROVED** [review]

**Pull request merged**